station/admin.py

```diff
@@ -0,0 +1,155 @@
+from django.contrib import admin
+from django.core.exceptions import PermissionDenied
+from django.db.models import Q
+from guardian.admin import GuardedModelAdmin
+from guardian.shortcuts import get_perms
+
+from .models import (
+    Basin,
+    Country,
+    Ecosystem,
+    Institution,
+    Place,
+    PlaceBasin,
+    Region,
+    Station,
+    StationType,
+)
+
+admin.site.site_header = "Paricia Administration - Stations"
+
+
+class BaseAdmin(GuardedModelAdmin):
```

Comment:
The `BaseAdmin` model might also be a good candidate to go to `management.admin` and then reuse it where needed.